landingzones/templates/landingzones/project_zones.html

```diff
@@ -161,6 +161,11 @@ <h3><i class="iconify" data-icon="mdi:briefcase-upload"></i> Landing Zones</h3>
     window.irodsShowChecksumCol = true;
   </script>
 
+  <!-- Superuser status variable -->
+  <script>
+    var isSuperuser = {{ superuser|lower }};
```

Comment:
We can simply do `{{ request.user.is_superuser }}` here, no need to set a separate variable in context.

Reply:
As a general note, in the future we'll want to replace things like this in a future with a user status ajax call which returns this info.

For anything serious, we don't want to provide this info to javascript in the template: this can be edited in run-time and "gain" superuser access. However, in this case it's OK, because the user can not really do anything dangerous with that info.